src/Products/Signature/Controllers/SignatureApiController.cs

```diff
@@ -1064,7 +1090,7 @@ private void SignText(string documentType, SignatureDataEntity signaturesData, S
         /// <param name="documentType">string</param>
         /// <param name="signsCollection">SignatureOptionsCollection</param>
         /// <param name="signer">SignatureSigner</param>
-        private void AddSignOptions(string documentType, SignatureOptionsCollection signsCollection, BaseSigner signer)
+        private void AddSignOptions(string documentType, List<SignOptions> signsCollection, BaseSigner signer)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make 'AddSignOptions' a static method.](https://app.codacy.com/gh/groupdocs-signature/GroupDocs.Signature-for-.NET-MVC/pullRequest?prid=5325146)